packages/knip/test/plugins/biome.test.ts

```diff
@@ -0,0 +1,18 @@
+import { test } from 'bun:test';
+import assert from 'node:assert/strict';
+import { main } from '../../src/index.js';
+import { resolve } from '../../src/util/path.js';
+import baseArguments from '../helpers/baseArguments.js';
+
+const cwd = resolve('fixtures/plugins/biome');
+
+test('Find dependencies with the biome plugin', async () => {
+  const { issues } = await main({
+    ...baseArguments,
+    cwd,
+  });
+
+  assert(issues.devDependencies['package.json']['@org/unused-config'])
+  assert(issues.unresolved['biome.json']['./shared/non-exist-base.json'])
+  assert(issues.unlisted['biome.json']['@org/unlisted-configs'])
```

Comment:
The idea of plugin tests is to have no issues left and `counters` with only `total` and `processed` (see other plugin tests for inspiration please).

Reply:
If we expect no issues, how do we test unused and unresolved dependencies which is the spirit of `knip` in the first place?

Reply:
@webpro I'm not sure if I'm misunderstanding something. I've skim through other tests and see lots of the tests assert on `issues` which is the same as this test. Can you help me understand this a bit more?

Reply:
Most of the plugin tests are pretty much "end to end" tests in that the fixture folder is consumed in its entirety including `package.json` that should contain the dependencies referenced in the tool configuration, and also executables. E.g. `node_modules/@biomejs/biome/package.json` contains `bin` field for `biome` binary that should actually be referenced and result in zero issues left. This full-circle approach results in well-defined and maintainable tests.

Testing that source code or configuration results in unused or unresolved can happen in additional/other tests. There is definitely room for improvement, but this is the direction I like to go.

Reply:
Ahh this is a really cool way of looking at testing. It was counter-intuitive to me at first but then it finally clicked. I updated the test. PTAL.